include/nfc/ndef/pair_msg.rst

```diff
@@ -0,0 +1,25 @@
+.. _nfc_pair:
+
+NFC Pair messages
```

Comment:
Is "NFC Pair message" a term? I couldn't find it anywhere.
If not, maybe we should combine this RST file with the ch.rst file into "Connection handover messages and records", as we do for URI and generic messages/records. That would require the files to be named more similarly though so it is clear that those two libraries are meant to be used together.

Reply:
I think we can merge documentation into one file and provide a code sample there. Yes this libraries in most cases will be used together

Reply:
What about changing name `pair_msg` -> `ch_msg`